python/lsst/sims/movingObjects/linearObs.py

```diff
@@ -0,0 +1,293 @@
+from __future__ import print_function, division
+import os
+import warnings
+import numpy as np
+
+from itertools import repeat
+from scipy import interpolate
+
+import lsst.sims.photUtils.Bandpass as Bandpass
+import lsst.sims.photUtils.Sed as Sed
+
+from lsst.sims.utils import haversine
```

Comment:
It was pointed out that no one knows what 'haversine' is.  `haversine` now wraps the more helpfully named `angularSeparation`.  Maybe call `angularSeparation` so that we can get rid of `haversine` if we ever want to?  Note that `angularSeparation` returns degrees.  `_angularSeparation` returns radians.